website/src/content/api-reference/commercetools-frontend-application-shell.mdx

```diff
@@ -73,6 +73,30 @@ export default EntryPoint;
 | `render`                         | `func`             |            (*)           | -       | The render function is called when the `<ApplicationShell>` is ready to render the actual application. This is the case when the required data (user, project) has been fetched and the application context has been initialized. It's **recommended to use the `children`** prop to benefit from a simpler setup.                                                                                                                                                                                                                                  |
 | `apolloClient`         | `ApolloClient`           |                       | -       | An optional instance of [ApolloClient](https://www.apollographql.com/docs/react/) to be used instead of the default one. This is usually the case when you need to configure the Apollo cache. See [`createApolloClient`](#createapolloclient).                                                                                                                                                                                                                                        |
 
+
+## ApplicationPageTitle
+
+Provides Custom Applications the ability to display a more useful page title based on the page location. The following format is used:
```

Comment:
nit: Perhaps we want to make it clear we mean browser tab title when we're referring to "page title" so users won't think about the title in the actual page content.